app/models/item_order.rb

```diff
@@ -12,4 +12,9 @@ class ItemOrder < ApplicationRecord
   def subtotal
     price * quantity
   end
+
+  def update_item_inventory
+    new_inventory = item.inventory - quantity
+    item.update_attributes(inventory: new_inventory)
+  end
```

Comment:
This should probably be renamed to something along the lines of "subtract_quantity_from_item" since this always subtracts quantity. When a user cancels an order, someone could quickly skim this file and think they can use this method to update it, when in reality it only ever takes inventory away from an item.

Reply:
@CoopTang  good point!

Reply:
`reduce_inventory` maybe

Reply:
@CoopTang @cjkelling @freeheeling this method name has been updated to #reduce_inventory